stomp_moveit/src/stomp_planner.cpp

```diff
@@ -177,7 +177,7 @@ bool StompPlanner::solve(planning_interface::MotionPlanDetailedResponse &res)
   using namespace stomp_core;
 
   // initializing response
-  res.description_.resize(1,"");
+  res.description_.resize(1,"plan");
```

Comment:
This change doesn't strictly belong in here so please say if I should remove it.
I added it while working with MoveIt benchmarks.

Reply:
It just adds a description tag that matches those used in OMPL and CHOMP so that it's easier to compare benchmark results.